auslib/web/admin/swagger/api.yaml

```diff
@@ -1231,6 +1231,36 @@ paths:
                   type: string
                 example: ["releng", "qa"]
 
+  /roles:
```

Comment:
Have you considered returning the list of users with each Role as part of this, too? It would make things simpler for the frontend, because you'd only need to make one HTTP request.

Reply:
Serves a better purpose than the two endpoints. Making changes.